### DISCUSSION TITLE
Why do hotkeys use option key?

### DISCUSSION THREAD

**Discussion by kenanmike** [SilentVoid13/Templater#312] [Help]
I'm wondering why the default hotkeys on this app are assigned to option key, when on a Mac this can only produce special characters and can't be used to perform menu shortcuts without the use of another key (like CMD or Ctrl). Is it simply because you use a Windows PC or is it something Obsidian assigned automatically? I know I can change each hotkey manually in Obsidian settings but might be one to change in the plugin default settings so that all Mac users don't have to do the same thing.
<img width="730" alt="Screenshot 2021-08-03 at 08 57 54" src="https://user-images.githubusercontent.com/85374001/127965456-8fd94fd0-0eb7-4ce4-aed6-3a3781331dfe.png">
<img width="712" alt="Screenshot 2021-08-03 at 08 58 02" src="https://user-images.githubusercontent.com/85374001/127965463-d370b19f-b556-4e1c-8b36-0f157458cc49.png">

**Comment by MichalBryxi** [comment]
- Can confirm that the default shortcuts (`option+n`) are broken on OSX
- Agree with the change to something else (maybe `cmd+option+[key]` would be enough?)
- There are numerous sources that try to elaborate [1](https://forum.obsidian.md/t/apple-shortcut-to-create-new-note-with-template-filled-in/26700), [2](https://forum.obsidian.md/t/hot-keys-nonfunctional/15243/2), [3](https://forum.obsidian.md/t/hotkeys-with-opt-alt-char-do-not-work-on-macos-they-insert-a-symbol-when-the-editor-is-active/72431/7) and according to [WhiteNoise](https://forum.obsidian.md/u/WhiteNoise) from obsidian (rephrasing here) "if option+key produces a symbol, it will **not** be picked up as shortcut **even if** you defined said shortcut inside obsidian":
> These hotkey will active the command when the editor is not active/focused. When the **editor** is **active**, they **will insert** the **symbol** instead. For the time being, we are gonna keep it this way.
- The sources above list various tricks how to solve this. The best one is to install different keyboard layout for OSX, which you IMO really don't want to do (adjusting core OS settings for one app? meh). And it's mostly a fake fix as for example the "Unicode Hex Input" fixes the `option+n` shortcut, but not `option+e` (as the latter produces a symbol and you end up where you started).
- So for any random passers: If you're on OSX, do not try to find a solution for default shortcuts and instead do-do change the shortcuts to something else (for example by adding `cmd` to the `option+key` combo).
